Components/List.js

```diff
@@ -13,7 +11,10 @@ const ListScreen = ({ navigation}) => {
           <ScrollView>
             {
             list.map((l, i) => {
+              // l is object at ith index in list from Movielist.js
```

Comment:
bad change. you should name your variables appropriately .... commetns only when u can't name it well.. you code shouldn't need docuymentation

Reply:
okk Sir